src/pages/RepoSearchPage/RepoSearchPage.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+
+import "antd/dist/antd.css";
+import Button from "@components/Button";
+import Input from "@components/Input";
+import RepoTile from "@components/RepoTile";
+import SearchIcon from "@components/SearchIcon";
+import { Layout, Row } from "antd";
+
+import GitHubStore from "../../store/GitHubStore";
+import { ApiResp, RepoItem } from "../../store/GitHubStore/types";
+import RepoBranchesDrawer from "./RepoBranchesDrawer";
+
+const { Content } = Layout;
+const gitHubStore = new GitHubStore();
+
+const RepoSearchPage = (): JSX.Element => {
+  const [repoList, setRepoList] = useState<RepoItem[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [currentValue, setCurrentValue] = useState<string>("");
+  const [currentRepo, setCurrentRepo] = useState<RepoItem | null>(null);
+  const [visibleDrawer, setVisibleDrawer] = useState<boolean>(false);
+
+  const getRepo = () => {
+    if (currentValue.length) {
+      setIsLoading(true);
+      gitHubStore
+        .getOrganizationReposList({
+          organizationName: currentValue,
+        })
+        .then((result: ApiResp<RepoItem[]>) => {
+          setRepoList(result.data);
+          setIsLoading(false);
+        });
+    }
+  };
+
+  const searchRepoOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    setCurrentValue(value);
+  };
+
+  const showDrawer = () => {
+    setVisibleDrawer(true);
+  };
+
+  const onClose = () => {
+    setVisibleDrawer(false);
+  };
+
+  const getRepoBranches = (repo: RepoItem): void => {
+    setCurrentRepo(repo);
+    showDrawer();
+  };
+
+  const getRepoTileJXS = () => {
```

Comment:
это плохо читаемо можно просто вставить сразу`
```suggestion
  <>
     {isLoading && <div>...Loading</div>}
     {!isLoading && repoList.map(repoItem => <Row justify="space-between" align="middle">
            <RepoTile RepoItem={repo} onClick={() => getRepoBranches(repo)} />
          </Row>)}
  </>
```